tests/migration_test.go

```diff
@@ -1926,7 +1924,7 @@ var _ = Describe("[Serial][rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][leve
 				Expect(err).ToNot(HaveOccurred())
 				tests.WaitForSuccessfulDataVolumeImport(dv, 600)
 				vmi := tests.NewRandomVMIWithDataVolume(dv.Name)
-				tests.AddUserData(vmi, "disk1", tests.GetFedoraToolsGuestAgentUserData())
+				tests.AddUserData(vmi, "disk1", "#!/bin/bash\n echo hello\n")
```

Comment:
ditto about echo hello

Reply:
Same here. I could probably remove the `echo`, but I want to execute "some" command.